index.js

```diff
@@ -0,0 +1,152 @@
+const express = require('express');
+const knex = require('knex');
+const knexConfig = require('./knexfile');
+const db = knex(knexConfig.development);
+const server = express();
+server.use(express.json());
+
+// PROJECTS
+
+//GET
+
+server.get("/projects", (req, res) => {
+    db("projects")
+      .then(projects => {
+          res.status(200).json(projects)
+      })
+      .catch(err => {
+          res.status(500).json(err)
+      });
+  });
+
+  server.get("/projects/:id", (req, res) => {
+    const { id } = req.params;
+    db("projects")
+      .where({ id: id})
+      .then(projects => {
+          res.status(200).json(projects)
+      })
```

Comment:
It looks like you don't get your specific project's actions here. That's part of MVP! Once you do that, you'll be good to go.